relay-prosperoconv/build.rs

```diff
@@ -0,0 +1,52 @@
+use std::fs;
+use std::path::Path;
+use std::process::Command;
+
+fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
+    println!("cargo:rerun-if-env-changed=RUSTFLAGS");
+
+    if !cfg!(sentry) {
+        return;
+    }
+
+    if Path::new("src/event.rs").exists() {
```

Comment:
As discussed:
- check for an empty `lib.rs`
- read the sha from somewhere (Cargo.toml, prosperoconv.version, ..)
- make it windows compatible, dir copy and tempdir